src/Umbraco.Web/Extensions/UdiExtensions.cs

```diff
@@ -16,24 +17,20 @@ public static IPublishedContent ToPublishedContent(this Udi udi)
             var guidUdi = udi as GuidUdi;
             if (guidUdi == null) return null;
 
+            var umbracoContext = UmbracoContext.Current;
+            if (umbracoContext == null) return null;
```

Comment:
I've added this guard, as a missing `UmbracoContext` would otherwise result in an exception. As this method also returns `null` when the `udi` isn't of type `GuidUdi`, I guess this shouldn't be a problem.